src/inmanta/moduletool.py

```diff
@@ -1336,6 +1336,7 @@ def release(
             gitprovider.tag(repo=module_dir, tag=str(release_tag))
             # bump to the next dev version
             self.release(dev=True, message="Bump version to next development version", patch=True)
+            print(f"Tag created successfully: {release_tag}")
```

Comment:
Why did you put this print here and not right after the `gitprovider.tag()` call? This can cause the output to be ordered in a strange way when we decide produce more output in the future.

Reply:
I thought I was nice to have it at the end even if more things are printed to find it easily as we will need it. but indeed i didn't think of the exception scenario. I'll change it.